airflow/operators/python.py

```diff
@@ -548,9 +558,9 @@ class PythonVirtualenvOperator(_BasePythonVirtualenvOperator):
         "requirements file" as specified by pip.
     :param python_version: The Python version to run the virtual environment with. Note that
         both 2 and 2.7 are acceptable forms.
-    :param use_dill: Whether to use dill to serialize
-        the args and result (pickle is default). This allow more complex types
-        but requires you to include dill in your requirements.
+    :param use_cloudpickle: Whether to use cloudpickle to serialize
```

Comment:
Here -the thing is that `dill` should remain a viable and documented option. I do not think it should be `deprecated` - it should be **parallell** to cloudpickle, both should rely on optional extra/dependencies installed.

Reply:
Totally agree, thank you!
I have modified the code to follow this